Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Rifles/L24B_rifle.yml

```
@@ -0,0 +1,118 @@
+- type: entity
+  parent: [ CMBaseWeaponRifle, RMCBaseWeaponMagazineVisuals ]
+  id: WeaponRifleL24B
```

```suggestion
  id: RMCWeaponRifleL24B
```
please, add RMC prefixes to RMC related stuff.